src/objective-c/GRPCClient/private/GRPCWrappedCall.m

```
@@ -0,0 +1,341 @@
+    gpr_free(_details_capacity);
+    if (_handler) {
+      _handler([NSError grpc_errorFromStatus:&status]);
+    }
+  };
+}
+
+@end
+
+@implementation GRPCWrappedCall{
+  grpc_call *_call;
+  GRPCCompletionQueue *_queue;
+}
+
+- (instancetype)init {
+  return [self initWithChannel:nil method:nil host:nil];
+}
+
+- (instancetype)initWithChannel:(GRPCChannel *)channel
+                         method:(NSString *)method
+                           host:(NSString *)host {
+  if (!channel || !method || !host) {
+    [NSException raise:NSInvalidArgumentException
+                format:@"channel, method, and host cannot be nil."];
+  }
+  
+  if (self = [super init]) {
+    static dispatch_once_t initialization;
+    dispatch_once(&initialization, ^{
+      grpc_init();
+    });
+    
+    _queue = [GRPCCompletionQueue completionQueue];
+    _call = grpc_channel_create_call(channel.unmanagedChannel, _queue.unmanagedQueue,
+                                     method.UTF8String, host.UTF8String, gpr_inf_future);
+    if (_call == NULL) {
+      return nil;
+    }
+  }
+  return self;
+}
+
+- (void)startBatchWithOperations:(NSArray *)operations {
+  [self startBatchWithOperations:operations errorHandler:nil];
+}
+
+- (void)startBatchWithOperations:(NSArray *)operations errorHandler:(void (^)())errorHandler {
+  NSMutableArray *opProcessors = [NSMutableArray array];
+  size_t nops = operations.count;
+  grpc_op *ops_array = gpr_malloc(nops * sizeof(grpc_op));
+  size_t i = 0;
+  for (id op in operations) {
+    [op getOp:&ops_array[i]];
+    [opProcessors addObject:[op opProcessor]];
+  }
+  grpc_call_error error = grpc_call_start_batch(_call, ops_array, nops,
+                                                (__bridge_retained void *)(^(grpc_op_error error){
+    if (error != GRPC_OP_OK) {
+      if (errorHandler) {
+        errorHandler();
+      } else {
+        [NSException raise:@"Operation Exception" format:@"The batch failed with an unknown error"];
```

Should I just have it fail silently?